dist/bin/dotd

```diff
@@ -0,0 +1,114 @@
+#!/usr/bin/env bash
+
+if [ -z "$PROG_HOME" ] ; then
```

Comment:
@DarkDimius I didn't get your point. This piece of code is a reliable way to get the package home in order to run `source "$PROG_HOME/bin/common"`. `bin/dotr` and `bin/dotc` do exactly the same thing.